x-pack/qa/rolling-upgrade/src/test/resources/rest-api-spec/test/mixed_cluster/40_ml_datafeed_crud.yml

```diff
@@ -17,11 +17,11 @@
 
 ---
 "Test old cluster datafeed with aggs":
+  - requires:
+      test_runner_features: "warnings"
   - skip:
-      features: warnings
       #TODO remove skip when master is bumped to 9.0.0
-      version: "all"
-      reason: "If we hit the old node we get a warning. If we hit the new node, we don't"
+      awaits_fix: "If we hit the old node we get a warning. If we hit the new node, we don't"
```

Comment:
This also looks funny; a cluster-feature (which is shared by the whole cluster) should already work here?